lib/whitehall/publishing_api.rb

```diff
@@ -45,7 +45,7 @@ def self.unschedule(edition)
     # PolicyGroup. Once we are pushing those to the Publishing API, this method
     # will need updating to return just the English locale for those models.
     def self.locales_for(model_instance)
-      model_instance.translated_locales
+      model_instance.translated_locales || :en
```

Comment:
I don't believe this change is required. I tried removing it and it still runs just find on my machine. Both the objects you have added have translations and should respond to that method.